src/pages/error/BadRequest.js

```diff
@@ -11,22 +11,22 @@ import { styles } from './styles/bad-request.styles'
 
 const BadRequest = () => {
   const { t } = useTranslation()
-  const windowSize = useBreakpoints()
+  const { isDesktop, isTablet, isMobile } = useBreakpoints()
 
-  const sizesTypography = {
-    desktop: { title: 'h2', description: 'subtitle1' },
-    tablet: { title: 'h3', description: 'subtitle1' },
-    mobile: { title: 'h4', description: 'subtitle2' }
-  }
+  const desktop = { title: 'h2', description: 'subtitle1' }
+  const tablet = { title: 'h3', description: 'subtitle1' }
+  const mobile = { title: 'h4', description: 'subtitle2' }
+
+  const windowSize = (isDesktop && desktop) || (isTablet && tablet) || (isMobile && mobile)
```

Comment:
rename from windowSize to something more appropriate, because this is not windowSize